bin/metrics-redis-llen.rb

```diff
@@ -47,6 +47,12 @@ class RedisListLengthMetric < Sensu::Plugin::Metric::CLI::Graphite
          description: 'Redis list KEY to check',
          required: true
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.